pkg/cdi/cache_test.go

```diff
@@ -554,6 +558,7 @@ devices:
 							opts = append(opts, WithAutoRefresh(false))
 						}
 						cache, err = NewCache(opts...)
+						require.Nil(t, err)
```

Comment:
Nit: replace with:
```suggestion
						require.NoError(t, err)
```

Reply:
Thanks for pointing this out. Changed as suggested